source/directives/uiTreeNode.js

```diff
@@ -481,12 +481,22 @@
 
                 moveWithinTree =  (targetNode && targetNode.$treeScope && targetNode.$treeScope.$id && targetNode.$treeScope.$id === treeOfOrigin);
 
-                // move horizontal
-                if (moveWithinTree) {
+                /* (jcarter) Notes to developers:
+                *  pos.dirAx is either 0 or 1
+                *  1 means horizontal movement is happening
+                *  0 means vertical movement is happening
+                *
+                *  Not sure what pos.distAxX does and not going to look into it now.... :P
+                */
 
-                  pos.distAxX = 0;
+                // Move nodes up and down in nesting level.
+                if (moveWithinTree && pos.dirAx) {
+
+                  //no longer used but keeping because use was not 100% understood.
+                  //pos.distAxX = 0;
```

Comment:
Yeah I can do that, my concern was that it has a use we might discover down the road.